objects/src/accounts/storage.rs

```diff
@@ -71,11 +71,21 @@ impl AccountStorage {
         &self.slots
     }
 
-    /// Returns a reference to the Merkle store that backs the storage.
+    /// Returns a mutable reference to the sparse Merkle tree that backs the storage slots.
+    pub fn slots_mut(&mut self) -> &mut SimpleSmt {
+        &mut self.slots
+    }
+
+    /// Returns a mutable reference to the Merkle store that backs the storage.
```

Comment:
It returns an immutable reference.

Reply:
oops, thanks for catching that! fixed the comments